<launch>

  <arg name="gui" default="true" />
  <arg name="gpu" default="-1" />
  <arg name="INPUT_IMAGE" default="/usb_cam/image_raw" />

  <node name="hand_pose_estimation_2d"
        pkg="jsk_perception" type="hand_pose_estimation_2d.py"
        output="screen">
    <remap from="~input" to="$(arg INPUT_IMAGE)" />
    <rosparam subst_value="true">
      gpu: $(arg gpu)
      model_file: $(find jsk_perception)/trained_data/SRHandNet.pts
    </rosparam>
  </node>

  <group if="$(arg gui)" >
    <node name="$(anon image_view)" pkg="image_view" type="image_view"
          args="image:=/hand_pose_estimation_2d/output/vis">
    </node>
  </group>

</launch>
